project-koku/koku
# Fix ocp vm csv export formatting

## Jira Ticket

[COST-5627](https://issues.redhat.com/browse/COST-5627)

## Description

This change will fix the response formatting for OCP VM csv exports

## Testing

1. Checkout Branch
2. Restart Koku
3. load ocp vm data
4. try to export csv data from api `curl --location -g 'http://localhost:8000/api/cost-management/v1/reports/openshift/resources/virtual-machines/' -H 'accept: text/csv' -o output.csv`
5. Should actually have data in the exported csv

## Release Notes

```markdown
* [COST-5627](https://issues.redhat.com/browse/COST-5627) Fix OCP VM Export CSV
```
